src/status_im2/subs/activity_center.cljs

```diff
@@ -34,6 +34,23 @@
         vals
         (reduce + 0))))
 
+(re-frame/reg-sub
+ :activity-center/seen?
+ :<- [:activity-center]
+ (fn [activity-center]
+   (:seen? activity-center)))
+
+(re-frame/reg-sub
+ :activity-center/unread-indicator
+ :<- [:activity-center/seen?]
+ :<- [:activity-center/unread-count]
+ (fn [[seen? unread-count]]
+   (if (zero? unread-count)
```

Comment:
Just a small comment on this.

I usually avoid nested `if`s, this is not the case, but IMO when nesting them they easilty get hard to read.
In this case I'd use a `cond`:
```clojure
(cond 
  (zero? unread-count) :unread-indicator/none
  seen?                :unread-indicator/seen
  :else                :unread-indicator/new)
```

What do you think?

Reply:
Agreed! `cond`'s FTW